src/components/tests/GeoPushpinChart.spec.tsx

```diff
@@ -59,21 +59,21 @@ describe("Geo Pushpin Chart", () => {
             },
         },
     };
+    function renderShallowComponent(customProps: Partial<IGeoPushpinChartProps>) {
+        const defaultProps: IGeoPushpinChartProps = {
+            projectId: "geoPushpinChart",
+            sdk: factory({ domain: "example.com" }),
+            location,
+        };
+        return shallow(<GeoPushpinChart projectId="geoPushpinChart" {...defaultProps} {...customProps} />);
+    }
+
     it("should render with custom SDK", () => {
-        const wrapper = shallow(
-            <GeoPushpinChart
-                location={location}
-                projectId="geoPushpinChart"
-                size={size}
-                sdk={factory({ domain: "example.com" })}
-            />,
-        );
+        const wrapper = renderShallowComponent({});
```

Comment:
Please pass the `sdk` property in this test only, and remove `sdk` in `defaultProps` of `renderShallowComponent`.
Other tests don't need `sdk` property